src/libaktualizr/primary/aktualizr.cc

```diff
@@ -32,10 +33,158 @@ Aktualizr::Aktualizr(Config &config) : config_(config) {
   uptane_client_->initialize();
 }
 
+void Aktualizr::FinalizeInstall() {
+  // TODO: is it too much?
+  FetchMetadata();
```

Comment:
Ideally this wouldn't fetch, just put the manifest.

Reply:
> You think putManifest() should be a public method of SotaUptaneClient?

Maybe? It definitely is wrong to fetch metadata when we actually should be pushing it. And is the rest of the SotaUptaneClient public functions really asynchronous? I didn't think we were quite there yet.

Also, there used to be way more public functions of SotaUptaneClient. I made as many private as I could, and I still think most should be private, but if we have to punch one or two holes, that isn't so bad.